src/DataObjects/WorkflowDefinition.php

```diff
@@ -303,6 +303,7 @@ public function getCMSFields()
                         $items
                     )
                 );
+                $dd->setHasEmptyDefault(true);
```

Comment:
LGTM. Could you also switch 297 to use `DropdownField::create()` while you're at it?

Reply:
Sure. I also updated the array syntax ... while I was at it. ;)